fhir-search/src/main/java/com/ibm/fhir/search/SearchConstants.java

```diff
@@ -104,7 +107,7 @@ private SearchConstants() {
 
     // set as unmodifiable
     public static final List<String> SEARCH_SINGLETON_PARAMETER_NAMES =
-            Collections.unmodifiableList(Arrays.asList(SORT, COUNT, PAGE, SUMMARY, ELEMENTS, RESOURCE_TYPE));
+            Collections.unmodifiableList(Arrays.asList(SORT, COUNT, PAGE, SUMMARY, TOTAL, ELEMENTS, RESOURCE_TYPE));
```

Comment:
Set?

Reply:
Order doesn't matter, so I can change it to a Set.